src/components/cart/CartCard.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from "styled-components";
+import { Product } from "../../types/Product";
+import { Counter } from "../product/Counter";
+import { useCartItem } from "../../hooks/useCartItem";
+import { DeleteShoppingCartIcon } from "../../assets/ShoppingCartIcons";
+import { Checkbox } from "../common/Checkbox";
+
+interface CartCardProps {
+  product: Product;
+  toggleSelectedItem?: (id: number) => void;
+  isSelected: (id: number) => boolean;
+  removeItemFromCartList: (id: number) => void;
+}
+
+export const CartCard = ({ product, toggleSelectedItem, isSelected, removeItemFromCartList }: CartCardProps) => {
+  const { id, name, imageUrl, price } = product;
+
+  const { changeCartItemQuantity, quantity } = useCartItem(id);
+  const handleSelectBoxChanged = () => {
+    if (toggleSelectedItem) toggleSelectedItem(id);
+  };
+
+  const handleDelete = () => {
+    removeItemFromCartList(id);
+  };
+
+  return (
+    <Style.Container>
+      <Style.ContentsOnReft>
```

Comment:
(사소)
```suggestion
      <Style.ContentsOnLeft>
```
typo 오류같아보여요.